transforms/geocable.py

```diff
@@ -645,19 +672,99 @@ def interpolate_rope(node1: Node, node2: Node, seg_count: int) -> List[Node]:
         for point in moveable
     ]
 
+def compute_binominal(n, k):
```

Comment:
```suggestion
def compute_binominal(n: int, k: int) -> float:
```

Reply:
Forgot to remove this function. The de_casteljau() does not need binomial.